RoddyCore/src/de/dkfz/roddy/config/RecursiveOverridableMapContainer.java

```diff
@@ -143,6 +143,10 @@ public V getValue(String id, V defaultValue) {
         return _getValueUnchecked(id);
     }
 
+    /**
+     * @throws RuntimeException if a value is requested that does not exist.
+     * @param id    Configuration value to return.
+     */
```

Comment:
1. Apparently, it is good practice to reuse official/existing core exceptions:
https://stackoverflow.com/questions/10016550/java-best-practices-when-throwing-exceptions-throwing-core-java-exceptions
2. There is https://docs.oracle.com/javase/7/docs/api/javax/xml/datatype/DatatypeConfigurationException.html, but the "Datatype..." does not fit. I would not use it here.
3. Maybe https://docs.oracle.com/javase/7/docs/api/java/text/ParseException.html, but I would not use it, because the configuration tree is not only build by parsing.

That's what I could find. I would make a new ConfigurationException.